lib/manager/components/version/FeedVersionNavigator.js

```diff
@@ -133,12 +135,56 @@ export default class FeedVersionNavigator extends Component<Props, State> {
     }
   }
 
+  _onSelectComparedVersion = (index: number) => {
+    if (index !== this.props.comparedVersionIndex) {
+      this.props.setComparedVersionIndex(this.props.feedSource, index)
+    }
+  }
+
+  /** Show comparison dropdown if there are 2+ versions to compare. */
+  _renderComparisonDropdown = () => {
+    const {
+      comparedVersionIndex,
+      sortedVersions,
+      version
+    } = this.props
+
+    if (sortedVersions.length >= 2) {
+      const comparedFeedVersion = sortedVersions[comparedVersionIndex - 1]
+      const compareTitle = comparedFeedVersion ? `Comparing to Version ${comparedVersionIndex}` : 'Compare'
+
+      // This custom formatter adds and hides the active version in the list to preserve version indexes.
```

Comment:
This seems like a UI hack. Perhaps it'd better to not pass the compareVersionIndex around and calculate it right here instead.

Reply:
I removed `comparedVersionIndex` in https://github.com/ibi-group/datatools-ui/pull/528/commits/03695eda1153b68d93751b1f0d27738edc2b41b1.